lib/packwerk/cli.rb

```diff
@@ -5,6 +5,10 @@
 
 module Packwerk
   # A command-line interface to Packwerk.
+  # Would it be better to refactor this to use `thor`, which is already in the Gemfile.lock
+  # via railties and other gems?
+  # That would allow the `format` option to be simpler and would help us not need to reinvent the wheel
+  # submit a PR for this?
```

Comment:
Will do this in a separate PR.

Note the `format` flag is not implemented in this PR.